src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
+* {
+  box-sizing: border-box;
 }
```

Comment:
Don't use * selector (it impacts performance), better use ``html`` or ``body`` tag.

![image](https://github.com/mate-academy/layout_product-cards/assets/54860641/25b7f749-aa72-455b-b152-f3256c50e7e9)